src/Monolog/Handler.php

```diff
@@ -44,21 +46,23 @@ public function __construct(HubInterface $hub, $level = Logger::DEBUG, bool $bub
      */
     protected function write(array $record): void
     {
-        $payload = [
-            'level' => self::getSeverityFromLevel($record['level']),
-            'message' => $record['message'],
-            'logger' => 'monolog.' . $record['channel'],
-        ];
+        $event = Event::createEvent();
+
```

Comment:
You can remove this blank line as the statements below are related to the one above